depends/windowsstore/nasmw/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 2.8)
```

Comment:
why duplicate?

Reply:
no need extra dependencies folder for windows store the `windows` is used for both windows and windows store if `windowsstore` doesn't exist.

Reply:
Will remove windows store